posts/comparing-java-lts-releases.md

```diff
@@ -0,0 +1,128 @@
+---
+authors:
+- Dan Ferguson
+tags:
+- Java
+date: 2019-01-14T14:06:00.000Z
+title: "Comparing Java LTS Releases"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/01/MVP-1.png
+---
+
+# Introduction
```

Comment:
Still needs to be removed ;)

Reply:
I'm not sure what you mean, do you mean remove the title?  Or the Introduction header?  Github's UI isn't making it very clear to me lol

Reply:
Remove the title (`# Introduction`), `title: "Comparing Java LTS Releases"` will be used for that instead